variables.tf

```diff
@@ -91,6 +91,12 @@ variable "jira_version_tag" {
   default     = null
 }
 
+variable "jira_replica_count" {
+  description = "Number of Jira application nodes"
+  type        = number
+  default     = 1
+}
```

Comment:
It would be nice to add a validator for a range of `[0-max-pods#]`

Reply:
Good point! There is no upper limit and you can also scale the `replicaCount` to zero, so I have added only `>= 0` validation.